azure-pipelines.yml

```diff
@@ -86,15 +88,25 @@ steps:
     Contents: '*'
     TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
+# - task: Bash@3
+#   displayName: 'Upload to Blob'
+#   condition: and(succeeded(), eq(variables['WHEEL_PUBLISH_FLAG'], 'true'))
+#   inputs:
+#     targetType: 'inline'
+#     script: |
+#       export AZCOPY_SPA_CLIENT_SECRET=$(AZURESDKPARTNERDROPS-SERVICE-PRINCIPAL-KEY)
+#       azcopy login --service-principal --application-id $(APPID)
+#       azcopy copy "$(Build.ArtifactStagingDirectory)/*"  "https://azuresdkpartnerdrops.blob.core.windows.net/drops/azure-iot-edge-tools-iotedgedev/pypi/$(VERSION)/"
+
 - task: Bash@3
-  displayName: 'Uplaod to Blob'
+  displayName: 'Upload to Blob'
   condition: and(succeeded(), eq(variables['WHEEL_PUBLISH_FLAG'], 'true'))
   inputs:
     targetType: 'inline'
     script: |
       export AZCOPY_SPA_CLIENT_SECRET=$(AZURESDKPARTNERDROPS-SERVICE-PRINCIPAL-KEY)
       azcopy login --service-principal --application-id $(APPID)
-      azcopy copy "$(Build.ArtifactStagingDirectory)/*"  "https://azuresdkpartnerdrops.blob.core.windows.net/drops/azure-iot-edge-tools-iotedgedev/pypi/$(VERSION)/"
+      azcopy copy "$(Build.ArtifactStagingDirectory)/*"  "https://sadevicelabsetup.blob.core.windows.net/test//azure-iot-edge-tools-iotedgedev/pypi/$(VERSION)/"
```

Comment:
We used 'python' for IoTEdgeHubDev. Is 'pypi' the better option? Either way, I think we should be consistent.

Reply:
yes, I have updated to python.